ed25519/shamir_test.go

```diff
@@ -0,0 +1,385 @@
+package ed25519_test
+
+import (
+	"math/rand"
+	"testing"
+	"time"
+
+	
```

Comment:
go format would make this just one space, not two.

Reply:
Executed go fmt.